src/internal/pjs/api_server.go

```diff
@@ -156,7 +143,7 @@ func (a *apiServer) ProcessQueue(srv pjs.API_ProcessQueueServer) (retErr error)
 			inputs = append(inputs, filesetID.HexString())
 		}
 		if err := srv.Send(&pjs.ProcessQueueResponse{
-			Context: "context is not implemented",
+			Context: hex.EncodeToString(jobCtx.Token),
```

Comment:
It would be nice if Token returned an object that had a String() method on it, so that someone doesn't decide here that it's hex.EncodeToString and someone in the future decides it's base64.EncodeToString.

Reply:
I think the String() function actually prints the hash to avoid leaking the context token.

Reply:
I see.  That's fine.  In the event that someone does `log.Proto(<job>)` and you don't want the token to be logged, be sure to annotate it in the proto file with `[(log.mask) = true];`.  (There is also (log.half) if you want to only log half the string.  We use that for OIDC tokens.)

Reply:
Oh yeah, good idea!